libbeat/docs/security/users.asciidoc

```diff
@@ -185,71 +185,73 @@ endif::serverless[]
 [[privileges-to-publish-events]]
 ==== Grant privileges and roles needed for publishing
 
-Users who publish events to {es} need to create and read from {beatname_uc}
+Users who publish events to {es} need to create and write to {beatname_uc}
 indices. To minimize the privileges required by the writer role, you can use the
-<<privileges-to-setup-beats,setup role>> to pre-load dependencies. Then turn off
-setup options in the  {beatname_uc} config file before running {beatname_uc} to
-publish events. For example:
+<<privileges-to-setup-beats,setup role>> to pre-load dependencies.
 
 ifndef::no_ilm[]
-[source,yaml]
-----
-setup.template.enabled: false
-setup.ilm.check_exists: false
-setup.ilm.overwrite: false <1>
-----
-<1> Omit `ilm.check_exists` and `ilm.overwrite` if ILM is disabled.
-endif::no_ilm[]
+When using ILM, turn off the ILM setup check in the {beatname_uc} config file before
+running {beatname_uc} to publish events:
 
-ifdef::no_ilm[]
 [source,yaml]
 ----
-setup.template.enabled: false
+setup.ilm.check_exists: false
 ----
 endif::no_ilm[]
 
 To grant the required privileges:
 
-. Create a *writer role*, called something like +{beat_default_index_prefix}_writer+, that has
-the following privileges (this list assumes the setup options shown earlier are
-set to `false`):
+. Create a *writer role*, called something like +{beat_default_index_prefix}_writer+,
+that has the following privileges:
++
+NOTE: The `monitor` cluster privilege and the `create` index privilege on
++{beat_default_index_prefix}-*+ indices are required in every configuration.
 +
 [options="header"]
 |====
-|Privileges | Why needed?
+|Type | Privilege | Purpose
```

Comment:
I would probably put the Privilege col before Type because I expect the most important info to be in the first col. But I can also see the value of putting the type first since that's how the UI is organized. It's up to you. I like having the extra col. In the past, 3-col tables were highly discouraged because they sometimes caused text overruns. I don't think this will be an issue with the new site design, tho, and I don't see any overruns in this topic, so I say let's do it!

Reply:
Make sure you apply this change to other tables that show privileges.

Reply:
Done. I stayed with putting the type first - experimented with putting it second, but unfortunately, it sort of "flows" into the third column:
<img width="698" alt="Screen Shot 2019-10-07 at 10 10 42 copy" src="https://user-images.githubusercontent.com/694597/66325392-15c4fc80-e91f-11e9-9fef-5d43601f0f79.png">

Reply:
It's unfortunate when tool limitations (not being able to control col width) drive content decisions, but I agree with you.